assets/paths.js

```diff
@@ -0,0 +1,85 @@
+// paths.js
+// configuration for gulpfile.js
+
+var paths = {};
+
+// Config
+paths.config = {};
+paths.config.minify_suffix = '.min';
+
+
+// Directories
+paths.directories = {};
+paths.directories.build = 'build/';
+
+
+
+//
+// Tasks
+//
+paths.tasks = {
+  css: {
+    default: {
+      source: 'css/main.css',
+      destination: 'style.css',
+      tailwind_config: 'tailwind.js',
+      watch: [
+        'css/**/*.{less,css}',
+        'tailwind.js',
+      ],
+    },
+  },
+  purge: {
+    default: {
+      source: paths.directories.build + 'style.css',
+      destination: 'style.purged.css',
+      config: {
+        content: [
+          '../templates/**/*.{twig,html}',
+          paths.directories.build + '**/*.js',
+        ],
+        extractors: [{
+          extractor: class TailwindExtractor {
+            static extract(content) {
+              return content.match(/[A-Za-z0-9-_:\/]+/g) || [];
+            }
+          },
+          extensions: ["html", "twig", "js"],
+        }],
+      },
+      watch: [
+        paths.directories.build + 'style.css',
+        paths.directories.build + '**/*.js',
+        '../templates/**/*.{twig,html}',
+      ],
+      watch_config: {
+        ignored: '../templates/_manifest.json',
+      },
+    },
+  },
+  js: {
+    urgent: {
+      source: [
+      ],
+      destination: 'urgent.js',
+      watch: [
+      ],
+    },
+    deferred: {
+      source: [
+      ],
```

Comment:
Add the PJAX files to the `deferred` task.

Reply:
Added. Note that IMHO this should have been on the sync-js branch.